mirrord/protocol/src/codec.rs

```diff
@@ -126,6 +128,8 @@ pub enum FileResponse {
     OpenDir(RemoteResult<OpenDirResponse>),
     GetDEnts64(RemoteResult<GetDEnts64Response>),
     ReadLink(RemoteResult<ReadLinkFileResponse>),
+    MakeDir(RemoteResult<MakeDirResponse>),
+    MakeDirAt(RemoteResult<MakeDirAtResponse>),
```

Comment:
New variants must be added at the end of enum definition here (due to how our serialization works) ;)